pubspec.yaml

```
@@ -24,7 +24,7 @@ dependencies:
   w_common: ^1.13.0
   w_flux: ^2.9.5
   platform_detect: ^1.3.4
-  quiver: ">=0.25.0 <1.0.0"
+  quiver: "^2.0.0"
```

Thanks for the PR! Would you be opposed to leaving the lower bound as-is? It seems we still have some internal dependencies on old versions that wouldn't resolve with this range... 😓 
```suggestion
  quiver: ">=0.25.0 <3.0.0"
```